test/parser/parser-util.test.ts

```diff
@@ -298,6 +298,7 @@ describe('Parser utils', () => {
                         },
                     },
                     scripts: [],
+                    customStyles: [],
```

Comment:
It's optional right?

Reply:
Yes it is. I used the same structure as the scripts in the parser-utils. https://github.com/WoodWing/studio-component-set-tools/pull/105/files#diff-b3b353126cc79046a828b28e511fd0a613b94cfd96a7ff84a18ea8f69b8f7ab4R48 So the parsed component definition always contains the property with an empty array. Is this what we want?

Reply:
This is fine.